src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import TodoContainer from './components/TodoContainer';
+import './App.css';
+// eslint-disable-next-line
+ReactDOM.render(
+  <React.StrictMode>
+    <Router>
+      <TodoContainer />
+    </Router>
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
```

Comment:
Please consider fixing this error which displays on the console. 

![Screenshot 2023-01-17 143809](https://user-images.githubusercontent.com/116313214/212913419-b083d4a5-499b-4c5b-8f70-588ea49afdf3.png)
This can be fixed by changing this line 👍